thirdeye/thirdeye-frontend/app/styles/components/rootcause-metric.scss

```diff
@@ -31,6 +31,7 @@
   }
 
   &__card {
+    height: 130px;
```

Comment:
did you try this with metrics with very long names, e.g. ones related to page views?

Reply:
Yes :)